### PR TITLE
adding links and explanations, metadata and flags

### PR DESCRIPTION
**Pull request by eftekhari-mhs**

### CODE REVIEW

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### License
+
+Eurostat has a policy of encouraging free re-use of its data, both for non-commercial and commercial purposes. 
+
+The license is available online at [here](https://ec.europa.eu/eurostat/about/policies/copyright).
+
+### Dataset Documentation and Relevant Links 
+
+- Documentation: <documentation_url>
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:53]: If no documentation, can remove this section.

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## About the Dataset
+
+### Download URL
+
+[TSV] file is available for download from <url>.
+
+### Overview
+The metadata for Population change - Demographic balance and crude rates at regional level (NUTS 3) is available online at [here](https://ec.europa.eu/eurostat/cache/metadata/en/demo_r_gind3_esms.htm)
+
+NUTS (Nomenclature of Territorial Units for Statistics) is a hierarchical classification for dividing regions of Europe. NUTSi, starting at NUTS1, is the ith classification which is more detailed than the NUTS(i-1). We’re importing population density for regions in the NUTS3 classification.
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:17]: I think we can just link to a page about NUTS after spelling out the acronym, and then list the P0/P1 datasets along with their covered geos (country to NUTS2, or country to NUTS3).

**Comment by tjann** [comment]
Hi @eftekhari-mhs -- is this ready for another round of review? I am still seeing "XXX" in the title, so maybe you're still working on this?

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+> This README is a template of what an import's README might look like.
+
+Author: eftekhari-mhs
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:11]: These are broken links (subsections don't exist?

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Eurostat has a policy of encouraging free re-use of its data, both for non-commercial and commercial purposes. 
+
+The license is available online at [here](https://ec.europa.eu/eurostat/about/policies/copyright).
+
+
+
+### Import Procedure
+
+> For this section, imagine walking someone through the procedure of
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:52]: Can rm this sentence

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+regenerating all artifacts in sequential order.
+
+#### Pre-Processing Validation
+
+> Include any steps, checks, and scripts you used to validate the source data.
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:57]: remove

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+> While writing script tests help make sure processing outputs are as expected,
+  also describe any steps, checks, and scripts you used to validate the
+  resulting artifacts. Here is an example of what someone might have done to
+  validate the artifacts.
+
+- Wrote and ran
+  [csv_template_mcf_compatibility_checker.py](https://github.com/datacommonsorg/data/blob/master/scripts/istat/geos/csv_template_mcf_compatibility_checker.py)
+  to validate that the resulting CSV and Template MCF artifacts are
+  compatible.
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:104]: Remove this section? I think this is just copied from Lijuan's

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+  python3 plot_samples.py
+  ```
+
+Automated validation:
+1. In the processing script (next section), there is an assert to check that
+  all expected columns exist in the CSV.
+
+#### Processing Steps
+
+> Include any commands for running your scripts. This is especially relevant if
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:76]: Can rm

Comment (tjann) [reply scripts/eurostat/regional_statistics_by_nuts/README.md:76]: Actually this section should also be removed if it was just leftover from the template.

File: scripts/eurostat/regional_statistics_by_nuts/README.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+### License
+
+Eurostat has a policy of encouraging free re-use of its data, both for non-commercial and commercial purposes. 
+
+The license is available online at [here](https://ec.europa.eu/eurostat/about/policies/copyright).
+
+
+
+### Import Procedure
```
Comment (tjann) [inline scripts/eurostat/regional_statistics_by_nuts/README.md:50]: I think I am realizing this whole section is just copied from the example? Is that right?

**Review by tjann — CHANGES_REQUESTED** [review]

File: scripts/eurostat/regional_statistics_by_nuts/README.md (hunk 8 above)
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/README.md:50]: yes, we should remove it.

File: scripts/eurostat/regional_statistics_by_nuts/README.md (hunk 3 above)
Comment (eftekhari-mhs) [reply scripts/eurostat/regional_statistics_by_nuts/README.md:11]: yes, looks like we have one section.

**Comment by eftekhari-mhs** [comment]
Unknown repo.

**Pull request closed without merging**